serve.sh

```diff
@@ -1 +1 @@
-bundler exec jekyll serve --future
\ No newline at end of file
+bundler exec jekyll serve --future --incremental
```

Comment:
no problems with leaving it in, we updated jekyll versions at some point, maybe they fixed it